api/src/main/java/io/strimzi/api/kafka/model/KafkaUserSpec.java

```diff
@@ -52,6 +53,16 @@ public void setAuthorization(KafkaUserAuthorization authorization) {
         this.authorization = authorization;
     }
 
+    @Description("Sets quotas on requests to control the broker resources used by clients.")
```

Comment:
This seems to mention only the requets percentage quota but not the consuemr or producer byte rates.